src/Augurk.CommandLine/Options/SharedOptions.cs

```diff
@@ -17,5 +12,29 @@ internal class SharedOptions
         /// </summary>
         [Option("url", HelpText = "URL to the Augurk Instance to which the features files should be published.", Required = true)]
         public string AugurkUrl { get; set; }
+
+        /// <summary>
+        /// Flag to indicate that the tool must run under integrated security to access the Augurk API's.
+        /// </summary>
+        [Option("useIntegratedSecurity", HelpText = "Use integrated security to access the Augurk API's. Do not specify username and password when using integrated security", MutuallyExclusiveSet = "username,password,useBasicAuthentication", Required = false)]
+        public bool UseIntegratedSecurity { get; set; }
+
+        /// <summary>
+        /// Flag to indicate that the tool must use basic HTTP authentication to access the Augurk API's.
+        /// </summary>
+        [Option("useBasicAuthentication", HelpText = "Use basic HTTP authentication to access the Augurk API's. You must also specify a username and (optionally) a password.", MutuallyExclusiveSet = "useIntegratedSecurity", Required = false)]
+        public bool UseBasicAuthentication { get; set; }
+
+        /// <summary>
+        /// Username for basic HTTP authentication against the Augurk API's.
+        /// </summary>
+        [Option("username", HelpText = "Username for basic authentication against the Augurk API's.", Required = false)]
+        public string BasicAuthnUsername { get; set; }
```

Comment:
Any reason whu Authentication can't be fully written? Why use Authn?
Especially since it is inconsistent with the MSBuild task.